From 3d118f688e8bf28f28ca5f454834b28bf31436a2 Mon Sep 17 00:00:00 2001
From: Ansgar Burchardt <ansgar@debian.org>
Date: Thu, 8 Sep 2016 17:36:18 +0200
Subject: [PATCH 5/6] Use CODENAME to identify suites which support merged-/usr

Unlike suite names (stable, testing, ...), the codename from the Release
file does not change meaning over time. So the codename should be used
for any hacks that should only happen for some releases.
---
 scripts/sid | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/scripts/sid b/scripts/sid
index 5866569..428c676 100644
--- a/scripts/sid
+++ b/scripts/sid
@@ -41,9 +41,8 @@ work_out_debs () {
 }
 
 first_stage_install () {
-	case $SUITE in
+	case "$CODENAME" in
 		etch|etch-m68k|jessie|lenny|squeeze|wheezy) ;;
-		oldstable|stable) ;;
 		*) setup_merged_usr ;;
 	esac
 
-- 
2.9.3

